kustomize-functions/validate-asm/image/main.go

```diff
@@ -253,23 +261,35 @@ func contains(s []string, e string) bool {
 	return false
 }
 
-func validateNodeCountAtLeast(r *yaml.RNode, meta yaml.ResourceMeta, min int, errorInfo string) error {
-	nodeCountAbsent, err := nodeAbsentOrAtLeast(r, meta, min, errorInfo, "spec", "nodeCount")
+func aggregateVCPUCount(r *yaml.RNode, meta yaml.ResourceMeta, vcpu int, total *int) error {
+	nodeCountAbsent, err := nodeAbsentOrAggregate(r, meta, vcpu, total, "spec", "nodeCount")
+	if err != nil {
+		return err
+	}
 	if nodeCountAbsent {
-		if _, err := nodeAbsentOrAtLeast(r, meta, min, errorInfo, "spec", "initialNodeCount"); err != nil {
+		if _, err := nodeAbsentOrAggregate(r, meta, vcpu, total, "spec", "initialNodeCount"); err != nil {
 			return err
 		}
+	}
+	return nil
+}
 
-	} else if err != nil {
-		return err
+func validateMinimalVCPUCount(count int) error {
+	if count < minimalTotalVCPUs {
+		return fmt.Errorf("the aggregated vCPU count is %d. " +
+				"Anthos Service Mesh recommends at least %d vCPUs. If you need to add nodes, " +
```

Comment:
We block on this in the installation script: https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/install_asm#L633

I thought this was a requirement, not a recommendation. Did this change recently?